src/actions/authors.js

```diff
@@ -0,0 +1,62 @@
+const { mysqlClient, contentfulManagement } = require('../support/config');
+const { pad, hashedSysId } = require('../support/utils');
+const { PersonEntry } = require('../models');
+
+const help = () => {
+  pad.log('Usage: npm run blog authors [ID]');
+};
+
+const createOne = async(id) => {
+  pad.log(`Creating entry for user: ${id}`);
+
+  const result = await mysqlClient.connection.execute(`
+    SELECT display_name, user_url, CONCAT(user_nicename, '@blog.europeana.eu') user_id
+    FROM wp_users WHERE ID=?
+  `, [id]);
+
+  const user = result[0][0];
+  const sysId = await hashedSysId(user.user_id);
+  const entry = new PersonEntry({ id: sysId });
+
+  entry.name = user.display_name;
+  entry.url = user.user_url;
+
+  await entry.createAndPublish();
+
+  return entry;
+};
+
+const createAll = async() => {
+  const result = await mysqlClient.connection.execute(`
+    SELECT DISTINCT post_author FROM blog.wp_posts WHERE post_type='post' ORDER BY post_author ASC
```

Comment:
`blog.wp_posts` is not necessary/won't work when the DB isn't called `blog`.
Just `wp_posts` should suffice.